beacon-chain/db/kv/schema.go

```diff
@@ -39,7 +39,7 @@ var (
 	// Specific item keys.
 	headBlockRootKey          = []byte("head-root")
 	genesisBlockRootKey       = []byte("genesis-root")
-	depositContractAddressKey = []byte("deposit-contract")
+	depositContractAddressKey = []byte("deposit")
```

Comment:
Wouldn't this require some kind of db migration?

Reply:
whoa, very nice find!